src/BuildIt.Forms/BuildIt.Forms.Controls/ContentButton.xaml.cs

```diff
@@ -223,6 +223,13 @@ private void OnCommandChanged()
                 if (IsPressed)
                 {
                     await VisualStateManager.GoToState(this, "Pressed");
+
+                    // Releasing the button after half a second. This should stop the button's visual state getting "stuck" in a ListView -RR
+                    await Task.Delay(500);
+                    if (VisualStateManager.GetVisualStateGroups(this)?.StateManager?.CurrentState("CommonStates") == "Pressed")
```

Comment:
Does the VisualStateManager not expose a CurrentState(this) as a short cut for this? If not, I should add it :-)

Reply:
Not that I could find